_conferences/mce-2018.md

```diff
@@ -6,5 +6,6 @@ location: Warsaw, Poland
 date_start: 2018-06-05
 date_end:   2018-06-06
 
+cfp_start: 2018-03-05
```

Comment:
Needs an end date

Reply:
Thanks. I've contacted organisers to ask for the end date. Couldn't find it anywhere online.